##### clip
roi_x_min: 4.0
roi_x_max: 10.0
roi_y_min: -20.0
roi_y_max: 20.0
roi_z_min: -1.75
roi_z_max: 2.0

vehicle_x_min: -1.2 #转换到车辆坐标系下后，将车身点云切除，车辆坐标系中心为后轴中心０点
vehicle_x_max: 3.5
vehicle_y_min: -1.0
vehicle_y_max: 1.0
vehicle_z_min: -0.5
vehicle_z_max: 2.5

# Enable pointcloud downsampling via VoxelGrid filter
is_downsample: true
# Downsampling leaf size
leaf_size: 0.1

##### remove ground
#1. the parameter of GetFilteredPclWithWindow
#  KWindowRows
#  KWindowCols
#  covariance_threshold_
#  height_of_hole
#2. the parameter of GetFilteredPclWithGrid
#  grid_width
#  grid_length
#  covariance_threshold_
#  max_ground_height_threshold
#3. the parameter of GetFilteredWithNearestNeighborVariance
#  covariance_threshold_
#  max_ground_height_threshold
#  min_dis_neighbour_points_threshold_
#4. the parameter of RemoveIsolatedPoints
#  min_dis_neighbour_points_threshold_
#  min_num_neighbour_points_threshold_
KWindowRows: 4
KWindowCols: 4
covariance_threshold_: 0.1
grid_width: 0.5
grid_length: 0.5
max_ground_height_threshold: 0.2
height_of_hole: -0.1
min_dis_neighbour_points_threshold_: 0.3
min_num_neighbour_points_threshold_: 3

##### cluster
###  euclidean_cluster
clusterTolerance: 0.5
minClusterSize: 4
maxClusterSize: 10000

### boundingBox
inEstimatePose: true
clusterMergeThreshold: 1.0

###  dbscan
min_sample: 2
eps: 4
bot: 0.4
top: 0.0
pic_height: 400
pic_width: 1200

##### tracker
name: "HmObjectTracker"
version: "1.1.0"
matcher_method: "HUNGARIAN_MATCHER"
filter_method: "KALMAN_FILTER"
track_cached_history_size_maximum: 10
track_consecutive_invisible_maximum: 2 #跟踪持续不可见次数
track_visible_ratio_minimum: 0.6
collect_age_minimum: 0 #存在的最短时间
collect_consecutive_invisible_maximum: 0 #收集跟踪结果持续不可见次数
acceleration_noise_maximum: 5
speed_noise_maximum: 0.4
match_distance_maximum: 4.0
location_distance_weight: 0.6
direction_distance_weight: 0.2
bbox_size_distance_weight: 0.1
point_num_distance_weight: 0.1
histogram_distance_weight: 0.5
histogram_bin_size: 10
use_adaptive: true
measurement_noise: 0.4
initial_velocity_noise: 5
xy_propagation_noise: 10
z_propagation_noise: 10
breakdown_threshold_maximum: 10.0
